.github/workflows/ubuntu_22_dpcpp.yml

```diff
@@ -0,0 +1,101 @@
+name: Linux (Ubuntu 22.04, Python 3.11, Intel DPC++ Compiler)
+on:
+  schedule:
+    # at 00:00 on Wednesday and Saturday
+    - cron: '0 0 * * 3,6'
```

Comment:
Does this need to be run on a bi-weekly basis?

Reply:
no